### PR TITLE
minor updates to foxy

### PR DESCRIPTION
**Pull request by flynneva**

### CODE REVIEW

File: nodes/usb_cam_node.cpp [hunk 1]
```diff
@@ -57,7 +57,7 @@ class UsbCamNode : public rclcpp::Node
   // shared image message
   sensor_msgs::msg::Image::SharedPtr img_;
   rclcpp::Publisher<sensor_msgs::msg::Image>::SharedPtr image_pub_;
-
+
```
Comment (flynneva) [inline nodes/usb_cam_node.cpp]:
```suggestion
```

Comment (flynneva) [reply nodes/usb_cam_node.cpp]: didnt mean to add a space here

**Comment by lucasw** [comment]
You should make your own fork the focus of your ros2 usb cam work and point others there, PR into the official repo from there.  I'm not using ros2 at all right now so I'm not right for reviewing or merging this.

(Maybe in six months or a year I'll evaluate ros2 again, I'll be really curious to see resource utilization comparisons between ros1 and ros2 usb camera implementations)

**Comment by flynneva** [comment]
got it. thanks for letting me know!

**Pull request closed without merging**